packages/ui/src/ui/pages/query-tracker/module/query_result/actions.ts

```diff
@@ -242,3 +250,35 @@ export function updateQueryResult(
         }
     };
 }
+
+export function loadQueryResultsErrors(
+    query: QueryItem,
+): ThunkAction<any, RootState, any, QueryResultsActions> {
+    return async (dispatch, getState) => {
+        const state = getState();
+        const {result_count} = query;
+
+        if (!result_count) return;
+
+        const inds: number[] = [];
+        for (let ind = 0; ind < result_count; ind++) {
+            if (!hasQueryResult(state, query.id, ind)) inds.push(ind);
+        }
+
+        if (inds.length === 0) return;
+
+        const resultsMeta = await dispatch(getQueryResultMetaList(query.id, inds));
```

Comment:
What about error handling?

Reply:
I've wrapped  call of `getQueryResultMetaList` with `wrapApiPromiseByToaster`, to show Network Errors etc in toast.

Also added splited error handling for failed sub-requests.